gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Mar 14 21:28:41 CET 2017
+#Wed Sep 16 22:00:43 CEST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
```

Comment:
When upgrading Gradle, you should actually run the `wrapper` task twice, so that `gradle-wrapper.jar` also gets updated (if there were any updates to it; but from version 3.4.1 to version 6.1.1 there certainly were updates).